pages/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { useState, FormEvent, ChangeEvent } from "react";
```

Comment:
Again, a lot of this code is ok as boilerplate but once you implement the GQL client and page shell I will be expecting a lot of this to be cleaned up!